src/pages/login/Login.js

```diff
@@ -0,0 +1,94 @@
+import React, {useState} from 'react';
+import {useNavigate} from "react-router-dom";
+import './Login.css';
+import user from "../../data/user.json";
+
+
+const Login = ({toggleIsAuthenticated}) => {
+
+    const [formState, setFormState] = useState({
+        email: "",
+        password: "",
+
+    });
+
+    const navigate = useNavigate()
+
+    function handleLogin() {
+
+        const field = user.find((log) => {
+            return log.emailadres === formState.email
+        })
+        console.log(field)
+
+        if (field.emailadres === formState.email && field.password === formState.password) {
```

Comment:
Als je niks invult en op inloggen drukt crasht de web 

TypeError: Cannot read properties of undefined (reading 'emailadres')